.github/workflows/release.yml

```
@@ -56,6 +56,33 @@
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+      - name: Verify tag matches Cargo.toml
+        run: |
+          set -euo pipefail
+          tag="${GITHUB_REF_NAME#v}"
+          toml_path="${CARGO_TOML_PATH:-Cargo.toml}"
+          cargo_version="$(
+            CARGO_TOML_PATH="$toml_path" \
+            "${{ steps.setup-python.outputs.python-path }}" - <<'PY' || echo ""
+import os, tomllib
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 72: could not find expected ':'

(syntax-check)